src/main/java/frc/robot/RobotContainer.java

```diff
@@ -156,11 +142,16 @@ public RobotContainer(RobotRunType runtimeType) {
                 climb = new Climber(new ClimberIO.Empty());
         }
 
+        new Trigger(() -> elevator.hightNotHome())
+            .onTrue(new InstantCommand(() -> swerve.setSpeedMultiplier(0.5)))
+            .onFalse(new InstantCommand(() -> swerve.setSpeedMultiplier(1)));
```

Comment:
Move the Trigger definition to the Elevator Subsystem. Then move the trigger binding to the `TriggerBindings` method